qa/archives/GNUmakefile

```diff
@@ -47,7 +47,8 @@ DEFLATE = small.0.xz small.meta small.index \
 	  pcp-pidstat-process-states.index pcp-pidstat-process-args.0.xz \
 	  pcp-pidstat-process-args.index pcp-pidstat-process-args.meta.xz \
 	  pcp-pidstat-timezone.0.xz pcp-pidstat-timezone.meta.xz \
-	  pcp-pidstat-timezone.index zero_disk_activity.index  \
+	  pcp-pidstat-timezone.index pcp-slabinfo.0 pcp-slabinfo.index \
```

Comment:
Could you compress these?  Just saves a little space and time on git clone/pull for everyone ... thanks.

Reply:
Resolved with commit  "pcp-slabinfo: review comment rework"